spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  describe "Iteration 2: Reading external data sets" do
+    it "exists" do
+      factory = VehicleFactory.new
+
+      expect(factory).to be_an_instance_of(VehicleFactory)
+    end
+    it "create vehicles using wa dmv data set" do
```

Comment:
Same thing here for your expect return values. Just make them datatypes vs the actual info. You will learn more about how to avoid this in Mod 3 using mocking and stubbing.